server/models/activity.model.js

```diff
@@ -0,0 +1,31 @@
+////////////// Import dependencies //////
+const mongoose = require('mongoose');
+// const bcrypt = require('bcrypt');
+const Schema = mongoose.Schema;
+////////////////////////////////////////
+
+
+//////// Create a database model //////
+const ActivitySchema = new Schema({
```

Comment:
doesnt the Activity also contain a post object? which is a list of post ids?

Reply:
I have already intimated that the post objects aren't added because we don't have a model for it yet; in the standup last week I told that I could go back and add post objects once we create a model for it

Reply:
you could still include in the schema but send a null value for now